Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -13,7 +13,7 @@ final class RewardsCollectionViewController: UICollectionViewController {
   // MARK: - Properties
 
   private let dataSource = RewardsCollectionViewDataSource()
-  private let viewModel = RewardsCollectionViewModel()
+  fileprivate let viewModel = RewardsCollectionViewModel()
```

Comment:
Why did the visibility change? 🤔

Reply:
Ah good catch! I'd forgotten that Swift 4 extended `private` access to include extensions in the same file 👍